pkg/appfile/parser.go

```diff
@@ -579,43 +183,54 @@ func GetOutputSecretNames(workloads *Workload) (string, error) {
 
 func parseWorkloadInsertSecretTo(ctx context.Context, c client.Client, namespace string, wl *Workload) ([]process.RequiredSecrets, error) {
```

Comment:
Hi @zzxwill 
This function is re-wrote. Instead of generating schema, we can get the secret name from CUE template comments through CUE SDK dircectly. Please help to review in case of my misunderstanding on this feature. Thx!

Reply:
That's better. Thanks.